walkthroughs/howto-multiple-listeners/README.md

```diff
@@ -39,6 +23,7 @@ Additionally, this walkthrough makes use of the unix command line utility `jq`.
     ```
 
     This command creates an Amazon EC2 Key Pair with name `color-app` and saves the private key at `~/.ssh/color-app.pem`.
+4. This walkthrough makes use of the unix command line utility `jq`. If you don't already have it, [you can install it from here](https://stedolan.github.io/jq/).
```

Comment:
Already used jq on step 3, should reorder 3 and 4

Reply:
Fixed